freeipa/src/main/java/com/sequenceiq/freeipa/service/image/ImageCatalogProvider.java

```diff
@@ -46,15 +46,26 @@
     @Value("#{'${cb.enabled.linux.types}'.split(',')}")
     private List<String> enabledLinuxTypes;
 
+    @Value("${cb.image.catalog.url}")
```

Comment:
This is already there in https://github.com/hortonworks/cloudbreak/blob/master/freeipa/src/main/java/com/sequenceiq/freeipa/service/image/ImageService.java#L63-L67

Reply:
hmm. I didn't intend to change anything related to the Image Catalog work. let me see if the diff is picking something up something due to rebasing issues.